examples/chat_with_document/main.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2023 The KubeAGI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/cors"
+	"github.com/tmc/langchaingo/schema"
+
+	zhipuaiembeddings "github.com/kubeagi/arcadia/pkg/embeddings/zhipuai"
+	"github.com/kubeagi/arcadia/pkg/llms/zhipuai"
+	"github.com/kubeagi/arcadia/pkg/vectorstores/chromadb"
+)
+
+var (
+	apiKey = flag.String("api-key", "", "used to connect to ZhiPuAI platform")
+	url    = flag.String("url", "", "the chromaDB scheme")
+	addr   = flag.String("addr", ":8800", "used to listen and serve GET request, default 8800")
+)
+
```

Comment:
use `corba` to provide this CLI with 
`chat start --apikey xxx --vector-store http://localhost:8000 --addr xxx